backend/editor/github_functions.py

```diff
@@ -21,8 +22,19 @@ def init_driver_and_repo(self):
         """
         Initalize connection to Github with an access token
         """
-        github_driver = Github(settings.access_token)
-        repo = github_driver.get_repo(settings.repo_uri)
+        access_token = settings.access_token
+        if not access_token:
+            raise HTTPException(
+                status_code=400,
+                detail="Access token is not set. Please add your access token in .env",
+            )
```

Comment:
is the access_token mandatory for all operations ?

I think it's only needed for the create pull request. I so, I would move this check to the create pull request method.

Reply:
It is also necessary to import a project. If we haven't specified it in the .env file, we can't effectively test the app as there is no data

Reply:
ok !